fastparquet/writer.py

```diff
@@ -571,6 +574,12 @@ def write(filename, data, partitions=[0], encoding="PLAIN",
                     fmd.row_groups.append(rg)
             fmd.num_rows = sum(rg.num_rows for rg in fmd.row_groups)
         else:
+            if write_index or write_index is None and not (
```

Comment:
I think that the compromise here should be that we continue to develop a minimal subset of features out to get to usability and then circle back around and pick up grouping things.  What's here works for writing pandas dataframes but not groupby objects.  I'd like to continue moving with this focus until things are fleshed out and working fully in that case.